Dockerfile

```diff
@@ -10,19 +12,19 @@ LABEL maintainer="gilbn"
 RUN \
   echo "**** install runtime packages ****" && \
   apk add --no-cache \
-    php8 \
-    php8-sqlite3 \
-    php8-mysqli \
-    php8-pdo_mysql \
-    php8-pdo_sqlite \
-    php8-gd \
-    php8-json \
-    php8-fileinfo \
-    php8-zip \
-    php8-ftp \
-    php8-ldap \
-    php8-tokenizer \
-    php8-intl && \
+    php81 \
+    php81-sqlite3 \
+    php81-mysqli \
+    php81-pdo_mysql \
+    php81-pdo_sqlite \
+    php81-gd \
+    php81-json \
```

Comment:
json is part of the main php81 package now.

Reply:
Resolved in https://github.com/linuxserver/docker-xbackbone/pull/21/commits/4a53c216ab533f905a6f1d80d57c942e298f5b36